frontend/src/views/ProfileView.vue

```diff
@@ -41,6 +41,12 @@ const logout = async () => {
 const editProfile = async () => {
   window.location.href = fxaEditProfileUrl;
 };
+
+// Load calendar and bookings information
+onMounted(async () => {
+  await calendarStore.fetch(call);
+  await appointmentStore.fetch(call);
+});
```

Comment:
This is a quick fix to fetch all necessary data. We might want to implement a store `init()` function that handles the fetch method and the initial data retrieval.

Reply:
Can we run these parallel?

Reply:
Haha of course! I'm a simple guy. I see an async function, I await. 😂